lib/abci/errors/NotSupportedAppVersionError.js

```diff
@@ -0,0 +1,17 @@
+const AbciError = require('./AbciError');
```

Comment:
We use ABCI errors to respond specific error code but BlockBeginResponse doesn’t have a code and do not affect block result, so we should just throw a normal domain error.

Reply:
Updated !